src/NLU.DevOps.CommandLine/EnumerableExtensions.cs

```diff
@@ -18,7 +18,7 @@ internal static class EnumerableExtensions
             }
 
             var indexedItems = items.Select((item, i) => new { Item = item, Index = i });
-            var results = new TResult[items.Count()];
+            var results = new List<TResult>();
```

Comment:
Comment to self (discussed in IM): By eliminating usage of `Enumerable.Count<T>()` here, we remove the need to enumerate the full lazy list to get the number of tests.
The list is enumerated below in the `foreach` at line 30.
The function to determine the set of test cases includes logic to parse the JSON, so by eliminating the extraneous enumeration, we only parse the |JSON once now.

Reply:
👍